site/content/en/docs/components/optional/status-reconciler.md

```diff
@@ -23,4 +23,4 @@ prow instance A, the jobs are not expected to be blindly lablled succeed by prow
 
 Note that `status-reconciler` is edge driven (not level driven) so it can't be used retrospectively.
 To update statuses that were stale before deploying `status-reconciler`,
-you can use the [`migratestatus`](https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus) tool.
+you can use the [`migratestatus`](https://github.com/kubernetes-sigs/prow/tree/main/maintenance/migratestatus) tool.
```

Comment:
The source code of migratestatus does not exist in either
- https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus or
- https://github.com/kubernetes-sigs/prow/tree/main/maintenance/migratestatus

Reply:
`migratestatus` was removed in this PR: https://github.com/kubernetes/test-infra/pull/32215
We can delete all references to it in the docs.